compiler/one-cmds/one-prepare-venv.aarch64

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
optional) maybe rename this to `one-prepare-venv.u2204-aarch64`?

Reply:
> Actually, this is not for ubuntu 22.04 but for ubuntu 18.04

Then maybe `one-prepare-venv.u1804-aarch64` ?

I assume supported python packages version may differ and for this, we may expect file contents may differ depending on Ubuntu version.

I'll check on my U22.04 RPi4 with this PR.